2021-03-19 17:21:39,586 INFO     
2021-03-19 17:21:39,588 INFO     Perform Semmle analysis...
2021-03-19 17:21:39,590 INFO     	Perform Semmle baseline analysis: True
2021-03-19 17:21:39,592 INFO     	Perform Semmle P10 analysis: True
2021-03-19 17:21:39,600 INFO     	Semmle Version: None
2021-03-19 17:21:39,604 INFO     	Changing directory: /opt/project/baseline_testcase/src
2021-03-19 17:21:39,608 INFO     
2021-03-19 17:21:39,611 INFO     	>> Executing command: make clean
2021-03-19 17:21:39,613 INFO     	>> From directory: /opt/project/baseline_testcase/src
2021-03-19 17:21:39,615 DEBUG    	Console output:
2021-03-19 17:21:39,631 DEBUG    		rm -rf *.o *.out CWE835
2021-03-19 17:21:39,646 INFO     	Changing directory: /opt/project/baseline_testcase
2021-03-19 17:21:39,651 INFO     
2021-03-19 17:21:39,653 INFO     	>> Executing command: do_semmle.create_project(/opt/project/baseline_testcase/.scrub/semmle_analysis, /opt/project/baseline_testcase, /opt/project/baseline_testcase/src, make all)
2021-03-19 17:21:39,656 INFO     	>> From directory: /opt/project/baseline_testcase
2021-03-19 17:21:39,670 INFO     
2021-03-19 17:21:39,673 INFO     	>> Executing command: odasa addLatestSnapshot --project /opt/project/baseline_testcase/.scrub/semmle_analysis 
2021-03-19 17:21:39,675 INFO     	>> From directory: /opt/project/baseline_testcase
2021-03-19 17:21:39,677 DEBUG    	Console output:
2021-03-19 17:21:39,682 DEBUG    		/bin/sh: odasa: command not found
2021-03-19 17:21:39,684 WARNING  Semmle analysis could not be performed. Please see log file /opt/project/baseline_testcase/.scrub/log_files/semmle.log for more information.
2021-03-19 17:21:39,690 WARNING  Traceback (most recent call last):
  File "/opt/project/tools/semmle/do_semmle.py", line 435, in run_analysis
    perform_analysis(tool_conf_data)
  File "/opt/project/tools/semmle/do_semmle.py", line 128, in perform_analysis
    add_snapshot(tool_conf_data.get('semmle_env'), tool_conf_data.get('semmle_addsnapshot_flags'))
  File "/opt/project/tools/semmle/do_semmle.py", line 314, in add_snapshot
    utils.scrub_utilities.execute_command(call_string, semmle_env)
  File "/opt/project/utils/scrub_utilities.py", line 159, in execute_command
    raise CommandExecutionError
utils.scrub_utilities.CommandExecutionError

